.gitignore

```diff
@@ -62,5 +62,10 @@ Scripts/tests/test_data/Results/*
 !Scripts/tests/test_data/Results/test/Matrices/uusimaa/time*
```

Comment:
I think `uusimaa` and `koko_suomi` matrices have same zones, so could have name for clarity.

Reply:
When running long-distance demand calculation through the UI, `submodel` is locked to "koko_suomi". I put it like this so that small-case UI tests can be easily performed.